packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -337,6 +338,35 @@ export async function getBookings({
       take: take + 1,
       skip,
     }),
+    prisma.booking.findMany({
+      where: {
+        OR: [
+          {
+            user: {
+              teams: {
+                some: {
+                  team: {
+                    isOrganization: true,
+                    members: {
+                      some: {
+                        userId: user.id,
+                        role: {
+                          in: ["ADMIN", "OWNER"],
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        ],
+        AND: [passedBookingsStatusFilter, ...filtersCombined],
+      },
+      orderBy,
+      take: take + 1,
```

Comment:
Oh wow it looks like we are taking/skipping on a per query basis and then concatenating the results, which could lead to weird behaviors.

Reply:
No worries. We'll take care of this oddity in another PR since all of the queries in this batch do the same thing. Thanks for your contribution!